slim_trees/utils.py

```diff
@@ -9,3 +14,28 @@ def check_version(version: str):
             f"Version mismatch: slim_trees version {slim_trees_version} "
             f"does not match version {version} of the model."
         )
+
+
+def pyarrow_table_to_bytes(table: pa.Table) -> bytes:
+    """
+    Given a pyarrow Table, return a .parquet file as bytes.
+    """
+    stream = pa.BufferOutputStream()
+    pq.write_table(
+        table, stream, compression="zstd", compression_level=8
```

Comment:
One idea for the future might be to try out different values here. Just checked and found out that disabling `zstd` here allows `lzma` to compress more later resulting in a slight overall improvement by around ~5% for the `trees_df` --> might generalize to other dataframes as well